### ISSUE TITLE
[8.0] [auth_admin_passkey] error if mail are disabled.

### ISSUE THREAD

**Issue by legalsylvain** [OCA/server-tools#754]
from #699, by @alexis-via 

## Step to reproduce

when you uncheck the 2 options to send mail on the configuration page, you get this error on authentification:

```
2017-02-28 15:12:11,589 12095 ERROR o8_test1 openerp.http: unexpected EOF while parsing
('', 0, 0, '')
Traceback (most recent call last):
  File "/home/alexis/new_boite/dev/odoo8/openerp/http.py", line 115, in dispatch_rpc
    result = dispatch(method, params)
  File "/home/alexis/new_boite/dev/odoo8/openerp/service/common.py", line 26, in dispatch
    return fn(*params)
  File "/home/alexis/new_boite/dev/odoo8/openerp/service/common.py", line 38, in exp_authenticate
    return res_users.authenticate(db, login, password, user_agent_env)
  File "/home/alexis/new_boite/dev/server-tools8/auth_admin_passkey/models/res_users.py", line 102, in authenticate
    self._send_email_passkey(cr, user_id, user_agent_env)
  File "/home/alexis/new_boite/dev/odoo8/openerp/api.py", line 268, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/alexis/new_boite/dev/odoo8/openerp/api.py", line 372, in old_api
    result = method(recs, *args, **kwargs)
  File "/home/alexis/new_boite/dev/server-tools8/auth_admin_passkey/models/res_users.py", line 34, in _send_email_passkey
    'True'))
  File "/home/alexis/new_boite/dev/odoo8/openerp/tools/safe_eval.py", line 312, in safe_eval
    c = test_expr(expr, _SAFE_OPCODES, mode=mode)
  File "/home/alexis/new_boite/dev/odoo8/openerp/tools/safe_eval.py", line 167, in test_expr
    code_obj = compile(expr, "", mode)
  File "<string>", line 0
    
    ^
SyntaxError: unexpected EOF while parsing
> /home/alexis/new_boite/dev/odoo8/openerp/tools/safe_eval.py(167)test_expr()
-> code_obj = compile(expr, "", mode)
```

**Comment by alexis-via** [comment]
Fixed in PR #752

**Comment by legalsylvain** [comment]
thanks.
